src/test/scala/mockws/MockWSTest.scala

```diff
@@ -20,6 +20,9 @@ import scala.concurrent.Future
 import scala.concurrent.duration._
 import org.scalatest.funsuite.AnyFunSuite
 import org.scalatest.matchers.should.Matchers
+import play.api.libs.ws.writeableOf_String
```

Comment:
These are implicit BodyParser implementations for reading and writting Strings to request bodies. They now need to be imported explicitly in Scala 3 (I think? Scala 3 compilation wouldn't work unless I did).

Reply:
I had the same "issue" in my projects, not sure if that's really intended though.